net/boinc/Makefile

```diff
@@ -0,0 +1,93 @@
+# Copyright 2020 by Christian Dreihsig and Steffen Möller
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=boinc
+PKG_VERSION:=7.16.5
+PKG_VERSION_SHORT:=$(shell echo $(PKG_VERSION)| cut -f1,2 -d.)
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/BOINC/boinc
+PKG_SOURCE_DATE:=2020-02-25
+PKG_SOURCE_VERSION:=fbdcc2d6165f0a3897341852f0fcf91a934724d0
+PKG_MIRROR_HASH:=f5ad881db7fafd6fc2c5360049d4f9c95c27d41641f5a77405d7df1514ecbc9f
+
+PKG_MAINTAINER:=
```

Comment:
Please add all three of you as maintainers.

Reply:
done